spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -98,10 +100,10 @@ def create_workspace_csv(workspaces, client_id_column)
     @config[:ads_client][:jdbc_url] = @ads.data['connectionUrl']
```

Comment:
warning: Similar code found in shared_contexts_for_lcm.rb:89, shared_contexts_for_lcm.rb:131 (mass = 32) (flay)